### PR TITLE
Drop type specs for L1Trigger

### PR DESCRIPTION
**Pull request by NiharSaha**
#### PR description:
Drop type specs in DQMOffline/L1Trigger python configuration files.

Central DQM migration campaign for drop type spces following
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1
and update the safer syntax for existing parameters like: (1) drop type specifications where the original parameter exists. (2) using "clone" instead of "deepcopy" (3) move all parameters inside the clone

#### PR validation:
Tested in CMSSW_12_3_X via runTheMatrix.py -l limited -i all --ibeos

### CODE REVIEW

**Comment by cecilecaillol** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: DQMOffline/L1Trigger/python/L1TEGammaEfficiency_cfi.py [hunk 1]
```diff
@@ -64,8 +64,7 @@
 )
 
 l1tEGammaEmuEfficiency = l1tEGammaEfficiency.clone(
-    subDirs=cms.untracked.vstring(
-        'L1TEMU/L1TObjects/L1TEGamma/L1TriggerVsReco'),
+    subDirs = ('L1TEMU/L1TObjects/L1TEGamma/L1TriggerVsReco',)
```
Comment (perrotta) [inline DQMOffline/L1Trigger/python/L1TEGammaEfficiency_cfi.py:67]:
```suggestion
    subDirs = ['L1TEMU/L1TObjects/L1TEGamma/L1TriggerVsReco']
```

File: DQMOffline/L1Trigger/python/L1TEtSumEfficiency_cfi.py [hunk 2]
```diff
@@ -31,8 +31,7 @@
 )
 
 l1tEtSumEmuEfficiency = l1tEtSumEfficiency.clone(
-    subDirs=cms.untracked.vstring(
-        'L1TEMU/L1TObjects/L1TEtSum/L1TriggerVsReco'),
+    subDirs = ('L1TEMU/L1TObjects/L1TEtSum/L1TriggerVsReco',)
```
Comment (perrotta) [inline DQMOffline/L1Trigger/python/L1TEtSumEfficiency_cfi.py:34]:
```suggestion
    subDirs = ['L1TEMU/L1TObjects/L1TEtSum/L1TriggerVsReco']
```

**Comment by cecilecaillol** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by cecilecaillol** [comment]
+l1

**Comment by perrotta** [comment]
+1

**Pull request merged**